### PR TITLE
feat: add checkbox component

### PR DESCRIPTION
**Pull request by Meet-Kalani**

### CODE REVIEW

File: src/components/ui/Checkbox/Checkbox.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+          }
+        }}
+        {...props}
+      />
+      <label htmlFor={id} className={style.label}>
+        {label}
+      </label>
+    </div>
+  );
+};
```
Comment (sanjaysah101) [inline src/components/ui/Checkbox/Checkbox.tsx:36]: Remove useRef, & optimize id. 

## Here the final components

```

import { FC, useId } from 'react';
import style from './checkbox.module.scss';

interface InputProps extends React.ComponentPropsWithRef<'input'> {
  label: string;
  checked?: boolean;
  disabled?: boolean;
}

const Checkbox: FC<InputProps> = ({ label, ...props }: InputProps) => {
  const id = `${useId()}-${label}`;

  return (
    <div className={style['checkbox-container']}>
      <input type="checkbox" className={style.checkbox} id={id} onChange={props.onChange} {...props} />
      <label htmlFor={id} className={style.label}>
        {label}
      </label>
    </div>
  );
};

export default Checkbox;

```

File: src/app/(home)/page.tsx [hunk 2]
```diff
@@ -4,12 +4,22 @@ import Hero from './_components/Hero';
...
+
   return (
     <>
       <NavBar />
       <Hero />
+      <Checkbox label="male" handleCheckboxInput={handleCheckboxInput} checked={isChecked} />
+      <Checkbox label="female" disabled />
       <Welcome />
       <GlobalEducation />
     </>
```
Comment (sanjaysah101) [inline src/app/(home)/page.tsx:25]: Remove Checkbox component from landing page.

File: src/styles/_constants.scss [hunk 3]
```diff
@@ -13,7 +13,7 @@ $color-text-primary-light: #43525f; /* Medium Grayish Blue */
 $color-text-primary: #2c3e50; /* Dark Grayish Blue */
 $color-transparent: transparent;
 $color-white: #fff; /* White */
-
+$color-disabled: #E5E5E6;/* Disable field color */
```
Comment (sanjaysah101) [inline src/styles/_constants.scss:16]: Rename `color-disabled `to `$color-gray-50`

File: src/components/ui/Checkbox/checkbox.svg
Comment (sanjaysah101) [inline src/components/ui/Checkbox/checkbox.svg:1]: Remove chekbox.svg if it is not required

**Review by sanjaysah101 — CHANGES_REQUESTED** [review]

**Review by sanjaysah101 — DISMISSED** [review]

**Review by ansopedia — APPROVED** [review]

**Review by sanjaysah101 — APPROVED** [review]

**Review by sanjaysah101 — APPROVED** [review]

**Review by ansopedia — APPROVED** [review]

**Comment by ansopedia** [comment]
Close #8

**Pull request merged**